### PR TITLE
Add 'resize' event on StreetView

### PR DESCRIPTION
**Pull request by tvernum**

### CODE REVIEW

**Comment by branflake2267** [comment]
Good idea. :)

File: gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java [hunk 1]
```diff
@@ -248,4 +248,10 @@ public final HandlerRegistration addVisibleChangeHandler(VisibleChangeMapHandler
         new VisibleChangeEventFormatter());
   }
 
+  /**
+   * Has to be called to resize the panorama if the canvas has changed
+   */
+  public final native void triggerResize() /*-{
+      $wnd.google.maps.event.trigger(this, 'resize');
```
Comment (branflake2267) [inline gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java]: 2 spaces

File: gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java [hunk 2]
```diff
@@ -248,4 +248,10 @@ public final HandlerRegistration addVisibleChangeHandler(VisibleChangeMapHandler
...
+  }-*/;
 }
```
Comment (branflake2267) [inline gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java:258]: add nl

File: gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaWidget.java [hunk 3]
```diff
@@ -245,4 +245,11 @@ public HandlerRegistration addResizeHandler(ResizeMapHandler handler) {
...
     return impl.addVisibleChangeHandler(handler);
   }
+
+  /**
+   * Has to be called to resize the panorama if the canvas has changed
+   */
+  public final void triggerResize() {
+    impl.triggerResize();
+  }
 }
```
Comment (branflake2267) [inline gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaWidget.java:256]: add nl

**Comment by branflake2267** [comment]
LGTM after nits :)

File: gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaWidget.java [hunk 4]
```diff
@@ -245,4 +245,11 @@ public HandlerRegistration addResizeHandler(ResizeMapHandler handler) {
   public HandlerRegistration addVisibleChangeHandler(VisibleChangeMapHandler handler) {
     return impl.addVisibleChangeHandler(handler);
   }
+
+  /**
+   * Has to be called to resize the panorama if the canvas has changed
```
Comment (branflake2267) [inline gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaWidget.java]: Could you add a period. :)

File: gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java [hunk 5]
```diff
@@ -248,4 +248,10 @@ public final HandlerRegistration addVisibleChangeHandler(VisibleChangeMapHandler
         new VisibleChangeEventFormatter());
   }
 
+  /**
+   * Has to be called to resize the panorama if the canvas has changed
```
Comment (branflake2267) [inline gwt-maps-api/src/main/java/com/google/gwt/maps/client/streetview/StreetViewPanoramaImpl.java]: Could you add a period. :)

**Comment by tvernum** [comment]
Heh... That's what I get for copying the existing code from MapWidget and not checking the formatting ;)

**Comment by branflake2267** [comment]
Good job. Thanks.

**Comment by tvernum** [comment]
I added the period on MapWidget too, cause I'm an accommodating person :)

**Comment by branflake2267** [comment]
Wahoo. :) Good job :)

**Comment by branflake2267** [comment]
LGTM :)

**Pull request merged**